radarr/anime-radarr.yml

```diff
@@ -1,8 +1,8 @@
-# Updated: 2023-07-19
+# Updated: 2023-08-01
 radarr:
-  radarr-anime:  # A custom name used to identify this particular instance.
-    base_url: Put your Radarr URL here
-    api_key: Put your API key here
+  anime-radarr:  # A custom name used to identify this particular instance.
+    base_url: [radarr_url_goes_here]
```

Comment:
I realize I'm late to the party here, but I just want to caution you on this syntax. `[]` is the syntax for arrays/sequences in YAML, so this change most likely will cause confusion if users just double-click the text between them, paste in a URL, and then save & run a sync without removing the square brackets.

Reply:
Thanks - will amend.